src/__tests__/Link.react.test.tsx

```diff
@@ -8,7 +8,7 @@ const { act } = renderer;
 test('Link changes the class when hovered', () => {
   const component = renderer.create(
     // eslint-disable-next-line jsx-a11y/anchor-is-valid
```

Comment:
I think it would be good if we had a different (less strict) set of rules for tests - but if you don't want to set that up now, we can do it in the unit test epic

Reply:
Good issue to add.